src/User/UserMetadataUpdater.php

```diff
@@ -49,7 +49,9 @@ public function whenPostWasPosted(Posted $event)
      */
     public function whenPostWasDeleted(PostDeleted $event)
     {
-        $this->updateCommentsCount($event->post, -1);
+        if (! isset($event->post->hide_time)) {
```

Comment:
There's no need to do an `isset` on any Eloquent Model property. These are all processed by `getAttribute` and if the property does not exist will return `null`. So here the value is either:

- `null`
- a `\Carbon\Carbon` or most likely `\Illuminate\Support\Carbon` object

Reply:
But these will go through [`__isset()`](https://github.com/laravel/framework/blob/39165ddea22cf837a34a8b01e4974cebb5c1af6d/src/Illuminate/Database/Eloquent/Model.php#L1480), which returns `false` if `getAttribute` returns `null`.